_sass/components/_image-example.scss

```diff
@@ -0,0 +1,3 @@
+.usa-image-example__figcaption {
```

Comment:
Can we add a comment here that this should be removed after the referenced ticket is complete, in case the ticket is deprioritized for whatever reason?

(I appreciate you adding the "Remove this file" task to that ticket! ❤️ )

```suggestion
// Remove after: https://cm-jira.usa.gov/browse/LG-11013
.usa-image-example__figcaption {
```